### PR TITLE
parallelize WPT tests run, add bandit linter, and misc changes

### PR DESCRIPTION
**Pull request by thiagowfx**
With the `--processes` this change:

- mapper-headless: [13min](https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/5204194529/jobs/9388094154?pr=828)
- chromedriver-headless: [15min](https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/5204194529/jobs/9388094278?pr=828)

Without this change:

- mapper-headless: [16min](https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/5189304483/jobs/9354117790)
- chromedriver-headless: [15min](https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/5189304483/jobs/9354117969)

The impact isn't that great, but it doesn't make things worse either.

### CODE REVIEW

File: examples/_helpers.py [hunk 1]
```diff
@@ -33,7 +33,8 @@ async def get_webdriver_session():
                                             "webSocketUrl": True
                                         }
                                     }
-                                }).json()
+                                },
+                                timeout=10).json()
```
Comment (thiagowfx) [inline examples/_helpers.py:37]: This change was suggested by bandit

File: runWPT.sh [hunk 2]
```diff
@@ -85,6 +86,10 @@ if [[ "$VERBOSE" == "true" ]]; then
     --log-mach -
     --log-mach-level info
   )
+elif [[ "$HEADLESS" == "true" ]]; then
+  WPT_RUN_ARGS+=(
```
Comment (sadym-chromium) [inline runWPT.sh:90]: Add comment here please

Comment (thiagowfx) [reply runWPT.sh:91]: Done

**Review by sadym-chromium — APPROVED** [review]
Ooops. I thought I pressed lgtm.

**Comment by OrKoN** [comment]
nproc is not available on Mac. Let's not write an shell scripts from now on in this repo, WDYT?

**Comment by thiagowfx** [comment]
x-ref https://github.com/GoogleChromeLabs/chromium-bidi/issues/876

**Pull request merged**